features/step_definitions/help_relinking.rb

```diff
@@ -17,15 +17,21 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #++
 
-Given /^the help link setting points to the project (.+)$/ do |project_name|
-  project_name.gsub!("\"", '')
-  p = Project.find_by_name(project_name)
+Given(/^the help link setting points to help page$/) do
   new_values = Setting.plugin_openproject_help_link
-  new_values['help_link_target'] = "/projects/#{p.identifier}"
+  new_values["help_link_target"] = OpenProject::Info.help_url
   Setting.plugin_openproject_help_link = new_values
 
   Redmine::MenuManager.map :top_menu do |menu|
     menu.delete :help
-    menu.push :help, OpenProject::Info.help_url, last: true
+    menu.push :help, OpenProject::Info.help_url, :last => true
   end
 end
+
+When(/^I visit the root page$/) do
+  visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.